CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Unreleased
 
+## Fixes
+
+### Updated task list template
+
+The task list pattern has been updated with changes to the [task list pattern in the Design System](https://design-system.service.gov.uk/patterns/task-list-pages/).
```

Comment:
I think it's worth explaining the change a bit here. Something like: "The [task list pattern has been updated in the Design System] to make it clearer which tasks users still need to do."